components/Buttons/src/MDCButton.m

```diff
@@ -314,6 +314,15 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return superSize;
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
+  [super traitCollectionDidChange:previousTraitCollection];
+
+  [self updateTitleFont];
```

Comment:
Please don't add this call to this PR. The PR should focus on adding the block support. If we need to adjust fonts after trait collection changes, please put that in a separate PR.

Reply:
yea. It's reasonable. Addressed in 6902a069263e11b222182e4e9c34461d7df68271